.github/workflows/release.yml

```diff
@@ -91,15 +92,53 @@ jobs:
           path: |
             build/dist/isle-*
 
+  flatpak:
+    name: "Flatpak (${{ matrix.arch }})"
+    runs-on: ${{ matrix.os }}
+
+    env:
+      APP_ID: pizza.isle.Isle
```

Comment:
I don't know the nature of the app IDs, but we generally use isledecomp as the org, so I imagine something like isledecomp.isle.Isle is better

Reply:
Sounds good! The full ID will look like `org.legoisland.Isle`, then.